exercises/practice/crypto-square/examples/success-standard/src/CryptoSquare.hs

```diff
@@ -6,10 +6,17 @@ import Data.List.Split (chunksOf)
 
 encode :: String -> String
 encode = unwords
+       . justify
        . transpose
        . (squareSize >>= chunksOf)
        . map toLower
        . filter isAlphaNum
   where
     squareSize :: String -> Int
     squareSize = ceiling . (sqrt :: Double -> Double) . fromIntegral . length
+    justify :: [String] -> [String]
+    justify [] = []
```

Comment:
I was indecisive on whether to add this. As written, this code would never pass an empty list to `justify`, so this case is unnecessary. However, I decided to add it to make `justify` potentially usable in other contexts, which seems like a fine idea. After all, the way `transpose` is used in `encode`, we would never pass an empty list to `transpose`, but `transpose` still would know how to handle one. In the same way, we can make `justify` know how to handle an empty list even if in the context of `encode` it will never need it.